addons/resource.language.en_gb/resources/strings.po

```diff
@@ -21588,3 +21588,22 @@ msgstr ""
 msgctxt "#39113"
 msgid "Center Mix Level in dB relative to metadata or default (-3 dB)"
 msgstr ""
+
+#. Label for "show information for unwatched items" setting option
+#: system/settings/settings.xml
+msgctxt "#39114"
+msgid "Episode thumb"
+msgstr ""
+
+#. Label for "show information for unwatched items" setting option
+#: system/settings/settings.xml
+msgctxt "#39115"
+msgid "Movie plot"
+msgstr ""
+
+#. Label for "show information for unwatched items" setting option
+#: system/settings/settings.xml
+msgctxt "#39116"
+msgid "Episode plot"
+msgstr ""
+
```

Comment:
translation script will remove this empty line

Reply:
Thought we need a newline at the end. Will remove

Reply:
yes one, but not two

Reply:
Thought there was only one. Made sure, there is now

Reply:
still two